tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es5",
+        "noEmit": true,
+        "noImplicitAny": false,
+        "sourceMap": false,
+        "moduleResolution": "node"
+    }
+}
```

Comment:
You will need if you want to test and check that it works, like you said before.